package.json

```diff
@@ -33,5 +33,8 @@
     "lerna-changelog": "^1.0.1",
     "npm": "^6.14.5",
     "typescript": "^3.9.2"
+  },
+  "dependencies": {
```

Comment:
We don't need sharp as a dependency for the root `package.json`. We should remove it here.